format-code.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+find firmware/common firmware/baseband firmware/application firmware/test -iname '*.h' -o -iname '*.hpp' -o -iname '*.cpp' -o -iname '*.c' | xargs clang-format -style=file -i
```

Comment:
Can you use firmware/test/applicaiton firmware/test/baseband so the test framework doesn't get formatted every time?

Reply:
gosh, that explains the speed... the lack of it :D tywm!